bin/tx-prover/src/proxy/mod.rs

```diff
@@ -129,7 +130,7 @@ impl LoadBalancerState {
             .iter()
             .map(|worker| Backend::new(worker))
             .collect::<Result<Vec<Backend>, _>>()
-            .map_err(|err| format!("Failed to create backend: {}", err))?;
+            .map_err(TxProverProxyError::BackendCreationFailed)?;
```

Comment:
Question: should we move instantiation of backends into the `Worker` constructor (i.e., `Worker::new()`)?

Reply:
I agree, that's better. I propose adding that task to [#1009](https://github.com/0xPolygonMiden/miden-base/issues/1009) or [#1001](https://github.com/0xPolygonMiden/miden-base/issues/1001). Is that ok?

Reply:
Yes - that's fine. Let's make a note about this in one of these issues.